.github/workflows/cloud-cleanup.yml

```diff
@@ -0,0 +1,54 @@
+name: Cleanup Cloud Resources
+env:
+  GO_VERSION: 1.21.3
+on:
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  build-eks-cleanup:
```

Comment:
Do we really need to split building and running into two jobs? Perhaps it can be the same to avoid the artifact thing.